packages/manager/apps/veeam-backup/src/components/Links/VCDOrgInfoLink.component.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
+import { LinkType, Links, OvhSubsidiary } from '@ovhcloud/manager-components';
+
+const urls: Partial<{ [key in OvhSubsidiary]: string }> = {
+  FR: 'https://www.ovhcloud.com/fr/lp/vmware-vcd-evolution/',
+  GB: 'https://www.ovhcloud.com/en/lp/vmware-vcd-evolution/',
+  DEFAULT: 'https://www.ovhcloud.com/en/lp/vmware-vcd-evolution/',
```

Comment:
As `GB` is the always default one, you can remove it

```suggestion
  DEFAULT: 'https://www.ovhcloud.com/en/lp/vmware-vcd-evolution/',
```